runelite-client/src/main/java/net/runelite/client/plugins/barrows/BarrowsBrotherSlainOverlay.java

```diff
@@ -54,41 +57,44 @@ private BarrowsBrotherSlainOverlay(Client client)
 	@Override
 	public Dimension render(Graphics2D graphics)
 	{
-		// Do not display overlay if potential is null/hidden
-		final Widget potential = client.getWidget(WidgetInfo.BARROWS_POTENTIAL);
-		if (potential == null || potential.isHidden())
+		if (config.showBarrowsOverlay())
```

Comment:
This should be a guard clause, not a large nested if statement.

Reply:
Changed it to guard clause.
Not sure whether the default barrows overlay can become movable, just adding toggle as requested by #7046.